crates/telio-dns/Cargo.toml

```diff
@@ -8,11 +8,10 @@ publish = false
 
 [dependencies]
 rand = "0.8"
-trust-dns-client = { git = "https://github.com/NordSecurity/trust-dns.git", tag = "v2.0.0" }
-trust-dns-proto = { git = "https://github.com/NordSecurity/trust-dns.git", tag = "v2.0.0" }
-trust-dns-resolver = { git = "https://github.com/NordSecurity/trust-dns.git", tag = "v2.0.0" }
-trust-dns-server = { git = "https://github.com/NordSecurity/trust-dns.git", tag = "v2.0.0", features = ["resolver"] }
-
+hickory-client = { git = "https://github.com/NordSecurity/trust-dns.git", tag = "v3.0.0-test" }
+hickory-proto = { git = "https://github.com/NordSecurity/trust-dns.git", tag = "v3.0.0-test" }
+hickory-resolver = { git = "https://github.com/NordSecurity/trust-dns.git", tag = "v3.0.0-test" }
+hickory-server = { git = "https://github.com/NordSecurity/trust-dns.git", tag = "v3.0.0-test", features = ["resolver"] }
```

Comment:
Just curious, what's with these name changes?

Reply:
The `trust-dns` author changed the crate name to `hickory`

Reply:
It's a nitpick but I would keep the old name here to make the diff smaller/simpler and make a follow up pr with a name change.

Reply:
Without the name change, it wouldn't find the package in the repo thus failing the compilation

Reply:
Is there a way we can work it around?

Reply:
You can rename dependency in `Cargo.toml` for example:
```bar = { git = "https://github.com/example/project.git", package = "foo" }```
However IMHO it isn't worth it